packages/dashboards/tests/integration/helpers/reportlize-test.js

```diff
@@ -0,0 +1,48 @@
+
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+const widgetModel = {
+  author: 'navi',
+  request: {
+    clone() {
+      return {
+        test: 'foo'
+      };
+    }
+  },
+  toJSON() {
+    return {
+      title: 'test',
+      visualization: {
+        type: 'line-chart'
+      },
+    }
+  }
+}
+
+moduleForComponent('helper:reportlize', 'Integration | Helper | reportlize', {
+  integration: true,
+});
+
+test('onModelChange', function (assert) {
+  assert.expect(2);
+
+  this.set('model', widgetModel);
+  this.render(hbs`<span class='request-content'>{{get (reportlize model) 'request.test'}}</span>`)
```

Comment:
this `span` isn't necessary is it? we can access the text without the span by just using `this.$().text().trim()`